zh.js

```diff
@@ -1,747 +1,747 @@
+windows.hat.header 选择头部 
+windows.back.header 选择背部饰品 
+windows.waist.header 选择腰部饰品 
+windows.face.header 选择表情
+windows.melee.header 选择近战武器 
+windows.attach.header 选择附件 
+windows.reticle.header 选择准星 
+windows.reticle.unlocked 于{0}级解锁 
+windows.reticle.req 需要{0}级 
+windows.dye.header 选择衣服 
+windows.dye.default 默认衣服 
+windows.dye.spin 衣服 
+windows.changelog.header 变更日志 
+windows.mods.community.header 社区模组 
+windows.mods.community.fav 最爱 
+windows.mods.community.new 最新模组 
+windows.mods.community.hot 最热门 
+windows.mods.community.name 模组名称 
+windows.mods.community.none 没有找到该模组 
+windows.publish.mod.header 发表模组 
+windows.publish.mod.thumbnail 没有缩略图 
+windows.publish.mod.name 模组名称 
+windows.publish.mod.paste 模组网址 
+windows.publish.mod.update 发表/更新 
+windows.publish.mod.login 以创建并上传! 
+windows.secondary.header 选择次要武器 
+windows.secondary.unlocked 已解锁 
+windows.secondary.req 需要至少{0}级 
+windows.advertise.header 广告 
+windows.support.header 支持 
+windows.players.header 玩家列表 
+windows.players.none 当前没有玩家... 
+windows.join.header 加入 
+windows.join.code 输入游戏 网址/代码 
+windows.client.header 客户端 
+windows.client.windows Windows游戏安装器 
+windows.client.mac macOS游戏安装器 
+windows.client.linux Linux游戏安装器 
+windows.ranked.header 排位 
+windows.ranked.login 以进行排位赛! 
+windows.ranked.level 您至少需要 <span color="black"> {0} 级</span> 以进行排位赛! 当前您的等级是 <span color="black"> {1} 级</span>. 
+windows.ranked.mode.r1v1 单排 
+windows.ranked.mode.r2v2 双排 
+windows.ranked.mode.r4v4 四排 
+windows.chall.header 挑战 
+windows.chall.login 登录以完成挑战! 
+thumbnail.upload.limit 失败. {0}kb 缩略图限制 
+image.upload.limit 失败. {0}kb 图片限制 
+thumbnail.upload.error 缩略图错误 
+mod.extracting 提取中 {0}/{1} 
+mod.disabled 模组已禁用 
+mod.error 模组错误! 
+mod.loading 加载模组中... 
+mod.invalid 无效模组文件! 
+matchmaker.full 满 
+matchmaker.full2 游戏已满. 
+matchmaker.updating 游戏更新中 
+matchmaker.invalid 没有找到游戏. 
+matchmaker.none 服务器已达上限 
+adblocker 禁用广告拦截器以获得免费的KR. 
+ad-failed 广告加载失败. 请稍后重试. 
+streamers.views {0} 观看者 
+streamers.none 当前没有Twitch直播! 
+error.extentions 如果您有任何(浏览器)插件, 请禁用它们 
+error.seek 点此以搜索新的游戏 
+custom.map.upload.limit 失败. 角色游戏数据限制为500kb 
+custom.map.missing 请选择至少一种游戏或地图 
+custom.mode.missing 请选择至少一种模式 
+custom.class.missing 请选择至少一种角色 
+server.config.maxPlayers 最大玩家数 
+server.config.minPlayers 最小玩家数 
+server.config.lives 生命数 
+server.config.gameTime 时间(分钟) 
+server.config.gravMlt 重力 
+server.config.tickRI 高传送率(High Tickrate) 
+server.config.maxPS 40 名玩家服务器 
+server.config.promServ 推广服务器 
+server.config.jumpMlt 跳跃力量 
+server.config.airStrf Air Strafing 
+server.config.deltaMlt 时间倍率 
+server.config.strafeSpd Strafe 速度 
+server.config.fiRat 武器射速 
+server.config.reSpd 换弹速度 
+server.config.slidTime Slide 时间 
+server.config.fixMov Fixed Movement
+server.config.slidSpd Slide Speed
+server.config.healthMlt 血量倍率 
+server.config.impulseMlt 武器后坐力 
+server.config.nameTeam1 队伍 1 名字 
+server.config.nameTeam2 队伍 2 名字 
+server.config.nameTeam3 队伍 3 名字
+server.config.nameTeam4 队伍 4 名字
+server.config.nameTeam5 队伍 5 名字
+server.config.t1Dmg 队伍 1 伤害 

```

Comment:
just need to fix whitespace on end of line

Reply:
umm idk what spaces do u mean but ill look into it

Reply:
oh, i see what you mean. i just cant edit anymore